source/partials/release_notes/_release-17-2-0.md.erb

```diff
@@ -0,0 +1,119 @@
+#### Authorization Plugins (beta)
+
+Currently GoCD provides multiple ways of Authentication like File based Authentication, LDAP/AD Authentication and Plugin Based Authentication. Apart from Authentication, GoCD provides ability to restrict certain operations to specific Users and Roles and manage them. Currently, roles can be managed only through GoCD and it does not provide an ability to use ACL’s defined in systems used for authentication. With this limitation, administrators need to configure ACL's in multiples places.
+
+With the introduction of Authorisation Plugin(Beta)  GoCD is moving in a direction where Authentication/Authorisation is moved out of GoCD into plugins. With this, the Authorisation Plugins would be responsible to provide Identity Service. The plugins have the flexibility to use any identity service providers like LDAP, Github etc.
+
+
+[LDAP Authentication plugin](https://github.com/gocd/gocd-ldap-authentication-plugin) is in place built on this Authorization endpoint. Please checkout this feature and provide us feedback to enhance this endpoint.
+
+#### Feature Enhancements
+
+* <%= link_to_commit 'c08843512b494ef1c483ebb699c615170dd13cf0', "Template Authorization UI and API enhancements" %>
+
+* No restriction on template view config. All pipeline group admins by default provided view permission to templates. Super admin can restrict this access.
```

Comment:
No restriction on template view config sounds a bit vague to me. Besides, it was implicitly already there for pipeline group admins. Can we just say `All pipeline group admins by default provided view permission to templates. Super admin can now restrict this access.`